src/scene_server.py

```diff
@@ -7,19 +7,35 @@
 from fastapi import FastAPI
 from lynx.common.object import Object
 from lynx.common.scene import Scene
+from lynx.common.enitity import Entity
 from pydantic import BaseModel
 
 
 def execution_runtime(pipe: AioConnection, object_id: int):
     from lynx.common.actions.move import Move
     from lynx.common.vector import Vector
+    from lynx.common.actions.action import Action
+
+    scene_serialized = pipe.recv()
+    scene: Scene= Scene.deserialize(scene_serialized)
+
+    def send(action: Action):
```

Comment:
Maybe `apply_to_scene` or `update scene` instead?

Reply:
But it does also receive scene with that action applied, right?

Reply:
I think that "send" would also be a better option. We only send the action, which is executed when the client sends the /tick request.